docs/HowToUseRamdump.md

```diff
@@ -0,0 +1,76 @@
+# How to Use Ramdump
+
+## Contents
+> [Ramdump Upload Steps](#ramdump-upload-steps)  
+> [Ramdump Parsing Steps](#ramdump-parsing-steps)  
+
+## Ramdump Upload Steps
+Below configuration must be enabled to support ramdump upload  
+1. Enable crashdump support (CONFIG_BOARD_CRASHDUMP=y)
+```
+Hardware Configuration > Board Selection -> Enable Board level logging of crash dumps to y
+```
+2. Enable ramdump (CONFIG_BOARD_RAMDUMP_UART=y)
+```
+Hardware Configuration > Board Selection -> Enable ramdump via UART to y
+```
+3. Enable Stack Dump (CONFIG_ARCH_STACKDUMP=y)
+```
+Hardware Configuration > Chip Selection -> Dump stack on assertions to y
+```
+4. Enable Debug (CONFIG_DEBUG=y)
+```
+Debug Options -> Enable Debug Output Features to y
+```
+5. Enable frame pointer for generating stack frame (CONFIG_FRAME_POINTER=y)
+```
+Debug Options -> Enable backtracking using Frame pointer register  to y
+```
+
+Run ramdump tool from host after target crashed.  
+1. If target got PANIC (asserted), below messages will be promted.
+```
+	****************************************************
+	Disconnect this serial terminal and Run Ramdump Tool
+	****************************************************
+```
+2. Disconnect/close serial termial (may be minicom)  
+
+3. Run ramdump tool
+```
+cd tools/ramdump/
+sh ramdump.sh
```

Comment:
Not ```./ramdump.sh``` ? The ramdump.sh is the bash, not sh.

Reply:
Ok